client/i2DX.css

```diff
@@ -4,6 +4,8 @@ body {
 	font: 15px Verdana;
 	color: #e9e8e7;
 	margin: 0; padding: 0;
+	overflow: hidden;
+	touch-action: pan-y;
```

Comment:
[NOTE]
In iOS 15's Mobile Safari, pinch-to-zoom functionality is enabled with this style, but for now, this is the interim solution.